ci/docker/install/docs_requirements

```diff
@@ -26,11 +26,11 @@ h5py==2.8.0rc1
 mock==2.0.0
 nose==1.3.7
 nose-timer==0.7.3
-numpy>1.16.0,<2.0.0
+numpy>1.16.0,<1.18.0
```

Comment:
Why is there a restriction on the numpy version? I think it has been fixed in master.

Reply:
I was told it was this PR's fix that helped resolve the issue https://github.com/apache/incubator-mxnet/issues/17154